Assembler.UnitTests/Assembler.UnitTests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
```

Comment:
Well for some reason I didnt manage to run the UTs, so I comment here in order to not forget to run them before I resolve and apprive merge